src/index.ts

```diff
@@ -1,10 +1,9 @@
-export * from './ast-types/mdast';
+export * from './ast-types';
 export * from './frontmatter';
 export * from './getJsonPathForPosition';
 export * from './getLocationForJsonPath';
 export * from './getters';
 export * from './parse';
-export * from './parseWithPointers';
```

Comment:
You probably didn't mean to commit the following change, did you?

Reply:
I did because it seems like an necessary thing to maintain as part of the public API - but meh who cares, adding it back.